app/presenters/formats/local_transaction_presenter.rb

```diff
@@ -69,6 +70,22 @@ def need_to_know
       }
     end
 
+    def all_devolved_administration_availabilities
+      {
+        scotland_availability: devolved_administration_availability(edition.scotland_availability),
+        wales_availability: devolved_administration_availability(edition.wales_availability),
+        northern_ireland_availability: devolved_administration_availability(edition.northern_ireland_availability),
+      }.compact
+    end
+
+    def devolved_administration_availability(availability)
+      if availability.type == "devolved_administration_service"
+        { type: "devolved_administration_service", alternative_url: availability.alternative_url }
+      elsif availability.type == "unavailable"
+        { type: "unavailable" }
+      end
```

Comment:
I reckon the linter is going to suggest you change these to:

``
case availability.type
when "devolved_administration_service"
```

once https://github.com/alphagov/publisher/pull/1457 is merged.

Reply:
Thanks, I updated.